### ISSUE TITLE
auth-user-pass error on linux

### ISSUE THREAD

**Issue by alexacallmebaka** [OpenVPN/openvpn3#117]
The problem I am encountering is that whenever I try to authenticate using the auth-user-pass option in openvpn3, I always get an error similar to this:

```
** ERROR ** org.gtk.GDBus.UnmappedGError.Quark._g_2dio_2derror_2dquark.Code36: GDBus.Error:net.openvpn.v3.sessions.error: Failed communicating with VPN backend: Failed calling D-Bus method Connect: GDBus.Error:net.openvpn.v3.backend.error.standard: Failed executing D-Bus call 'Connect': Configuration parsing failed: ERR_PROFILE_OPTION: option_error: remote option not specified
```
or just plain old:

``session-start: ** ERROR ** Failed to start session``

The relevant line form my ovpn file is:

``auth-user-pass creds.txt``

My VPN provider is Nord. My OS is Debian 10.

I have tried every way I can of fixing this, the autostarter, the legacy openvpn2 cli, etc. but I always get one of thise errors. Please help!

**Comment by dsommers** [comment]
This is not related to OpenVPN 3, but the OpenVPN 3 Linux client.  Please see the [related ticket](https://github.com/OpenVPN/openvpn3-linux/issues/19) for more information.

TL;DR: --auth-user-pass with credentials stored in a file is not supported.  But there exists functional workarounds.

**Comment by dsommers** [comment]
We have just released a workaround document on this feature: https://openvpn.net/openvpn-3-linux-and-auth-user-pass/
